templates/module/Cargo.toml

```diff
@@ -6,8 +6,13 @@ authors = ["{{authors}}"]
 resolver = "2"
 rust-version = "1.83"
 
-[dependencies]
-micro-rdk = {git = "https://github.com/viamrobotics/micro-rdk.git", features = ["{{mcu}}"], version = "0.4.0", rev = "380a098"}
+[dependencies.micro-rdk]
+git = "https://github.com/viamrobotics/micro-rdk.git"
+version = "0.4.1-rc1"
+rev = "v0.4.1-rc1"
```

Comment:
will these rev fields still be necessary? would be great if we can move away from this to just reference a specific version. I don't think I've ever seen another rust project that does this for their templates.

Reply:
I'd like that too, but it doesn't seem to be possible right now. This at least eliminates the need for the post-release BUMP commit, so I think it is a net win.

Reply:
I am interested in the answer to this question